ccloud/acl.sh

```diff
@@ -121,6 +121,14 @@ if [[ $? != 0 ]]; then
   exit 1
 fi
 
+echo -e "\n# Verify cluster is a Confluent Cloud Enterprise cluster"
+echo "ccloud kafka acl list"
+OUTPUT=$(ccloud kafka acl list 2>&1)
```

Comment:
@codyaray @DABH : is there a better way to check if a Kafka cluster is CCE or CCP?